suite/tests/linux/signal_pre_syscall.c

```diff
@@ -129,220 +186,231 @@ START_FILE
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
-        fdiv     v10.2d, v11.2d, v12.2d
+        fdiv     v0.2d, v0.2d, v1.2d
```

Comment:
Curious why this sequence of fdiv needed to be changed. Does it matter to the test what regs are used with the fdiv?